src/helpers.c

```diff
@@ -1250,3 +1251,63 @@ void warn_nosigcheck(const char *file)
 	warn("\n");
 	warn("THE SIGNATURE OF %s WILL NOT BE VERIFIED\n\n", file);
 }
+
+int write_into_file(char *file, char *content, char *mode)
```

Comment:
> What are the use cases for mode here? I think you can hardcode it to "w". Not sure if we are ever going to need "a".

If you don't think we will need the "append" I can hardcode to "w", and if we need the append at some point then we can add it then.

Reply:
> And I think you need the content size here. I think it's better than using strlen because not necessary the file is a text file.

I don't understand this comment, I'm not using strlen

Reply:
you are using sizeof() what's not working. I'm suggesting you not to fix it by using strlen()